docker/runtime/nodejs-6.10/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:6.10-alpine
```

Comment:
Note that the current available image for node is a development environment, it contains python, gcc, build tools... it is useful when building Node application but the result is a 480Mb image (vs current 82 Mb). We can revisit this once the prod node container is published.